slider/src/components/HorizontalSlider/HorizontalSlider.vue

```diff
@@ -240,7 +240,7 @@ $rail-height: 6px;
 $mark-height: 4px;
 
 .slider__container {
-  padding: 0px 20px;
+  padding: 10px 20px;
```

Comment:
Some letters, such as "g" need this. See the gif above where I hadn't added this vertical margin; the "g" is cut off.